src/main/java/uk/gov/ons/validation/common/Constants.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.ons.validation.common;
+
+public final class Constants {
+
+        public static final String WRANGLER_NAME = "WRANGLER_NAME";
```

Comment:
Formatting issues here. line 5 should be indented to the same level as the constructor below it.